### PR TITLE
Clarify how to contribute

### PR DESCRIPTION
**Pull request by serra**
This is my current understanding of how to contribute to the wiki.
If it makes sense, please consider to merge.

If it doesn't make sense, then we should make it more clear how to contribute, because I should be able to do this.

Cheers!

### CODE REVIEW

File: tiddlers/$__StoryList.tid [hunk 1]
```diff
@@ -1,6 +0,0 @@
-created: 20181111055700759
```
Comment (jamesgorman2) [inline tiddlers/$__StoryList.tid]: Looks like you've deleted the files while adding the ignores.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 node_modules
 package-lock.json
+favicon.ico
+$__plugins_felixhayashi_tiddlymap_misc_defaultViewHolder.tid
```
Comment (jamesgorman2) [inline .gitignore:5]: FYI, there are _probably_ more than this, TiddlyWiki has some funny behaviour with timestamps. Not worth doing anything about it yet though.

**Review by jamesgorman2 — CHANGES_REQUESTED** [review]
Thanks Marijn, 

the only change is to revert the deletion of the tiddlers that have been excluded. Otherwise looks good.

FYI, Pete and I were talking about how to script the save button so that it will do all the git things needed to make it easier for less technical coaches to make changes. The plan was to make sure that the script will revert any of these meaningless changes.

File: .gitignore (hunk 2 above)
Comment (serra) [reply .gitignore:5]: I'm not sure about the timestamps. It's a bit awkward that were using a version control system *and* use timestamps in the files.

File: tiddlers/$__StoryList.tid (hunk 1 above)
Comment (serra) [reply tiddlers/$__StoryList.tid]: Yes, I thought this was the quickest way to go.
I'll ignore them and mark them as assume-unchanged.

Comment (serra) [reply tiddlers/$__StoryList.tid]: And revert the delete.

**Comment by serra** [comment]
@jamesgorman2 I think this addresses your issues.

File: .gitignore (hunk 2 above)
Comment (jamesgorman2) [reply .gitignore:5]: Yes it is. Hopefully we can hide some of that in the interim. When Pete and I talked about this I raised that if you dig into fixing wiki editing too much it starts to dovetail with how the website works, and then into branding and UX and a whole bunch of other stuff that is not as important yet, so a stop-gap seems appropriate for now.

**Review by jamesgorman2 — APPROVED** [review]

**Pull request merged**